### PR TITLE
Revert cobbymed, stasis beds, and cloning removal when?

### PR DESCRIPTION
**Pull request by JJRcop**
got you!

the answer is "when you code it"

who wants to code that? weird

### CODE REVIEW

**Comment by Arturlang** [comment]
Good meme.

**Comment by TheVekter** [comment]
Okay, joking aside, the name's gotta change.

**Comment by XDTM** [comment]
Time to place an open radio next to the programmer and share this wisdom with everyone

**Comment by ATH1909** [comment]
Unironically merge this; it'd be funny.

File: code/modules/research/nanites/nanite_programmer.dm [hunk 1]
```diff
@@ -129,3 +130,9 @@
 				timer *= 10 //convert to deciseconds
 				program.timer_trigger_delay = timer
 			. = TRUE
+
+/obj/machinery/nanite_programmer/Hear(message, atom/movable/speaker, message_language, raw_message, radio_freq, list/spans, message_mode)
+	. = ..()
+	var/static/regex/when = regex("(?:^\\W*when|when\\W*$)", "i") //starts or ends with when
+	if(findtext(raw_message, when) && !istype(speaker, type))
```
Comment (ATH1909) [inline code/modules/research/nanites/nanite_programmer.dm]: this line makes sure that two different nanite programmers won't bounce this phrase back and forth between themselves, right?

Comment (JJRcop) [reply code/modules/research/nanites/nanite_programmer.dm]: actually in testing the same programmer would hear itself and recurse before runtiming on infinite loop, but it also covers that case too

Comment (Cyberboss) [reply code/modules/research/nanites/nanite_programmer.dm]: It wouldn't work if two different subtypes of `nanite_programmer` got together. Change `type` to `/obj/machinery/nanite_programmer` to fix that

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/research/nanites/nanite_programmer.dm (hunk 1 above)
Comment (JJRcop) [reply code/modules/research/nanites/nanite_programmer.dm]: I could completely cover it by blacklisting the phrase "When you code it!!", after putting it in a define of course

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**